Formula/naml.rb

```diff
@@ -0,0 +1,33 @@
+class Naml < Formula
+  desc "Convert Kubernetes YAML to Golang"
+  homepage "https://github.com/kris-nova/naml"
+  url "https://github.com/kris-nova/naml/archive/refs/tags/v1.0.3.tar.gz"
+  sha256 "0842633268b06be82db4dd10c3c938f756f613c44c15c2d935b933409da8c4bd"
+  license "Apache-2.0"
+  head "https://github.com/kris-nova/naml.git", branch: "main"
+
+  depends_on "go" => [:build, :test]
```

Comment:
Is Go needed at runtime?

Reply:
Yeah, but it does not need the latest go though.

Reply:
Actually, it is a good idea to have it as runtime dependency. Good call.